src/CommunityToolkit/Xamarin.CommunityToolkit/Converters/DateTimeInRangeConverter.shared.cs

```diff
@@ -0,0 +1,25 @@
+﻿using System;
+using System.Globalization;
+using Xamarin.CommunityToolkit.Extensions.Internals;
+using Xamarin.Forms;
+
+namespace Xamarin.CommunityToolkit.Converters
+{
+	public class DateTimeInRangeConverter : ValueConverterExtension, IValueConverter
```

Comment:
What is the purpose of this converter? why can't we use IsInRangeConverter?

Reply:
It was just to exemplify the concept of this converter. I have removed it.